src/networkDetails.ts

```diff
@@ -79,13 +79,13 @@ export const addresses: IAddresses = {
     FUSE_18_SOHM: "0x6eDa4b59BaC787933A4A21b65672539ceF6ec97b", // Olympus Pool Party
     FUSE_36_SOHM: "0x252d447c54F33e033AD04048baEAdE7628cB1274", // Fraximalist Money Market
     PT_TOKEN_ADDRESS: "0x0E930b8610229D74Da0A174626138Deb732cE6e9", // 33T token address, taken from `ticket` function on PRIZE_STRATEGY_ADDRESS
-    PT_PRIZE_POOL_ADDRESS: "0xEaB695A8F5a44f583003A8bC97d677880D528248", // NEW
-    PT_PRIZE_STRATEGY_ADDRESS: "0xf3d253257167c935f8C62A02AEaeBB24c9c5012a", // NEW
+    PT_PRIZE_POOL_ADDRESS: "0xEaB695A8F5a44f583003A8bC97d677880D528248",
+    PT_PRIZE_STRATEGY_ADDRESS: "0xf3d253257167c935f8C62A02AEaeBB24c9c5012a",
     ZAPPER_POOL_V1: "0x04f2694c8fcee23e8fd0dfea1d4f5bb8c352111f",
     BONDINGCALC_V2: "0x7b1a5649145143F4faD8504712ca9c614c3dA2Ae",
     TREASURY_V2: "0x9a315bdf513367c0377fb36545857d12e85813ef",
     FIATDAO_WSOHM_ADDRESS: "0xe98ae8cD25CDC06562c29231Db339d17D02Fd486",
-    GIVING_ADDRESS: "0x2604170762A1dD22BB4F96C963043Cd4FC358f18",
+    GIVING_ADDRESS: "0x42842Fd91c83B84D51D1be514B9716d62c61CbEF",
```

Comment:
I removed these & refactored the one place that they were still used to use the `addresses.ts`... so that we don't have them in 2 places. confusing. We need to clean up this file across the board still.

Anyways, in this change I'm about to push all Tyche addresses are in one spot now, rather than 2.